integrations/extensions/starter-kits/magnolia/README.md

```diff
@@ -0,0 +1,124 @@
+Sorry.  The search failed!  Please try again another time.
+
+![Search failed](./assets/search-failed.gif)<br>
+
+- Still in the "Search" action, add a "New Step".  In the new step:
+  - In "Assistant says" hit `$` and select "Ran Successfully" and then click on `</>` in the upper right of that box to see the full JSON for the response.  In there, you should see a field called `variable` with a value that looks something like `step_123_result_1`.  Copy that value.
+  - Click "abc" in the upper right and delete the variable in "Assistant says" (we only put it there to copy the variable name).
+  - Click the fX button to add variables and set the `search_results` variable to an `Expression`. Type in the following replacing `step_123_result_1` with the actual variable name:
+  
+	  ```
+	  search_results = ${step_123_result_1}.body.results
+	  ```
+  
+![Set result items](./assets/set-search-results.gif)<br>
+
+  - In "And then", select "Go to another action", and select the action "Show Search Results"
+ 
+
+- Close the action editor (by clicking X in the upper right)
+- Go to "Actions" > "Set by assistant" > "No action matches" and remove all the steps from the action.  Add in a new step.  Under "And then" select "Go to another action" and select "Search" and click "End this action after the subaction is completed".
+- You may also want to go to "Actions" > "Set by assistant" > "Fallback" and do the same thing as in the previous step.  Note, however, that this will prevent your assistant from escalating to a human agent when a customer asks to connect to a human agent (which is part of the default behavior for "Fallback") so only do this if you do not have your bot connected to a human agent chat service.  For more details on connecting to human agents within Watson Assistant see [our documentation](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-human-agent) and [blog post](https://medium.com/ibm-watson/bring-your-own-service-desk-to-watson-assistant-b39bc920075c).
+- Go to the Search action and remove "Search" from the "Customer starts with" list so that the search action _only_ triggers via the "Go to another action" settings described in steps 13-15 above.  If you skip this, then all actions will also be considered by the intent recognizer as a possible intent, which adds unnecessary complexity to the intent recognition and thus could result in lower overall intent recognition accuracy. 
+- Do the same as above for removing "Show Search Results" and "Show Search Result" from the corresponding actions' "Customer starts with" list
+
+## Using this Starter Kit
+
+Once this starter kit is properly installed, you can issue a query to your bot and if there is no other action that you've configured that matched that query then it will generate search results for that query.
```

Comment:
Would be nice to include a screenshot in this section too, e.g.:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/11746051/193292236-edff63c3-7b6d-4a8c-8cd6-178ee22c1958.png">